### PR TITLE
added sign-in and sign-up

### LINKED ISSUE #22: [fullstack] register login and logout functionality
Stateless Session - Using cookies?

Collection Name on MongoDB is 'users' and database is 'scrt-pwd-data'

### PR DESCRIPTION
**Pull request by floatingbug**
- added sign-in and sign-up.
- insert: `JWT_SECRET:"secret"` to .env

### CODE REVIEW

**Comment by corazoan** [comment]
1. i think you didn't add validation functionality in frontend.
2. For instances is email is valid or not, password at least 8 character long , contain a number , contain a special character.
3. Do we store user password as plain text or as hash.
(ignore any message if you add this functionality)

**Comment by floatingbug** [comment]
i added the following:
- storing user in db.
- validate credential (on sign up).
- sign in with jwt (jwt stores in local storage in browser.

the points you listed is not implemented yet, i will do it soon, tomorrow maybe, is that ok?
i think hashing the password is a good idea i can do this also tomorrow

**Comment by anonymous-sidhax** [comment]
Please check the issue - [#22 ](https://github.com/Collaborative-Open-Source-Projects/scrt-pwd-link/issues/22). 
- I have mentioned the database and table name to use. 
- password is stored as plain text
- On clicking login button -> it should redirect to /dashboard.
- token is not getting stored in local storage

Apart from this - I have added JWT_SECRET to vercel environment variables so that it will work on live :)

File: .env
Comment (anonymous-sidhax) [inline .env:1]: Why are we adding .env file on github? it is supposed to be for local only.

File: app/models/userModel.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import clientPromise from "@/app/lib/mongodb";
+
+export async function addUserModel(doc){
+	try{
+		const client = await clientPromise;
+	
+		const userColl = client.db("scrt-pwd-link").collection("user");
```
Comment (anonymous-sidhax) [inline app/models/userModel.js]: use correct database and table please

**Comment by floatingbug** [comment]
changes:
- database and collection name renamed.
- password is storing as hash now.
- redirect to dashboard if sign-in is successful.

if did nothing change with de storing of the token. if i sign-in the token is stored in the localstorage of the browser.

File: .env
Comment (anonymous-sidhax) [inline .env:1]: You don't need to add .env file when pushing your code

**Review by anonymous-sidhax — APPROVED** [review]

**Pull request merged**